torch/onnx/_internal/onnx_proto_utils.py

```diff
@@ -287,3 +291,21 @@ def _find_onnxscript_op(
                 else None,
             )
     return onnx_function_list, included_node_func
+
+
+def _update_option_with_kwargs(options, kwargs):
```

Comment:
Is this file the best place to put it?

Reply:
Do you have a better place? I was thinking of leaving it into verification.py, but it seems more of a utils